erlang-sd_notify.spec

```diff
@@ -12,14 +14,20 @@ Release:	9%{?dist}
 Summary:	Erlang interface to systemd notify subsystem
 Group:		Development/Languages
 License:	MIT
-URL:		https://github.com/%{upstream}/erlang-%{realname}
-%if 0%{?el7}%{?fedora}
-VCS:		scm:git:https://github.com/%{upstream}/erlang-%{realname}.git
-%endif
-Source0:	https://github.com/%{upstream}/erlang-%{realname}/archive/%{version}/erlang-%{realname}-%{version}.tar.gz
-Source1:	erlang-sd_notify-rebar.config
+URL:		https://github.com/lemenkov/erlang-sd_notify
+VCS:		scm:git:https://github.com/lemenkov/erlang-sd_notify.git
+Source0:	https://github.com/lemenkov/erlang-sd_notify/archive/%{version}/erlang-%{realname}-%{version}.tar.gz
+Source1:	sd_notify.app
```

Comment:
this was missing in PR, I grabbed it from CBS SRPM http://cbs.centos.org/kojifiles/packages/erlang-sd_notify/0.1/10.el7/src/erlang-sd_notify-0.1-10.el7.src.rpm